tox.ini

```diff
@@ -130,6 +146,11 @@ deps =
 [testenv:extra-deps-pinned]
 basepython = python3.8
 deps =
+    uvloop==0.19.0
+    zstandard==0.22.0
+    bpython==0.24
+    ipython==8.10.0
```

Comment:
These seem to be the latest versions. They should instead be the minimum versions for which tests pass.

Reply:
I know.

I was just making tests, because when I did like you said, the CI tests was not approved